src/libraries/System.Private.CoreLib/src/Internal/Runtime/CompilerServices/Unsafe.cs

```diff
@@ -10,11 +10,11 @@
 
 #pragma warning disable SA1121 // explicitly using type aliases instead of built-in types
 #if TARGET_64BIT
```

Comment:
Just wondering why this isn't testing things by just removing these aliases? I'd expect us to be able to just remove all of the usings and have things "just work"
If they don't, its likely a compiler bug we should flag @cston on and otherwise, we should be generally good to check things in.
The compiler just uses the built-in IL operators after all and we were already using aliases of the right size....

Reply:
I gave it a shot: https://github.com/dotnet/runtime/pull/36159 . Some places needed an extra unsafe or cast, but it was looked pretty straightforward otherwise.